src/components/intern-app.html

```diff
@@ -45,17 +52,43 @@
             type: Object,
             value: () => database
           },
-          profileToEdit: Object,
+          userData: Object,
+          editUser: {
+            type: Boolean,
+            value: false
+          },
+          users: {
+            type: Boolean,
+            value: true
+          },
+          newUser: {
+            type: Boolean,
+            value: false
+          }
         }
       }
 
-      ready() {
-        super.ready();
+      connectedCallback() {
+        super.connectedCallback();
         this.addEventListener('profileToEdit', this.profileToEdit);
+        this.addEventListener('deleteUser', this.deleteUser);
+      }
+
+      createNewUser() {
+        this.users = false;
+        this.newUser = true;
       }
 
       profileToEdit(e) {
-        this.profileToEdit = e.detail.profileToEdit;
+        this.userData = e.detail.profileToEdit;
+      }
+
+      show() {
+        this.userData === undefined ? (this.editUser = false) && (this.users = true) : (this.editUser = true) && (this.users = false);
```

Comment:
i'm not sure... does this work? crazy if it does.
You'd probably want to rework this, though, for readability
```suggestion
        this.userData === undefined ? 
          this.setProperties({'editUser': false, 'users': true}) : 
          this.setProperties({'editUser': true, 'users': false});
```
_I think the right function is `setProperties`, you may have to look into it_

Reply:
It does work but - wow! this suggestion is so much simpler!